README.md

```diff
@@ -64,43 +64,44 @@ $ rm -rf node_modules
 $ time npm ci
 ```
 
-time: `0m43.004s`
+time: `0m18.030s`
 
 ```
 $ rm -rf node_modules
 $ time yarn install
 ```
 
-time: `0m36.384s`
+time: `0m30.571s`
 
 Testing install speed **with** cache `node_modules` folder.
 
 ```
 $ time npm install
 ```
 
-time: `0m10.964s`
+time: `0m54.944s`
 
 ```
 $ time npm ci
 ```
 
-time: `0m46.317s`
+time: `0m21.810s`
 
 ```
 $ time yarn install
 ```
 
-time: `0m1.601s`
+time: `0m2.109s`
 
 ## Conclusion
 
-Date: 2018-03-08
+Date: 2018-10-02
 
 Yarn is so faster than npm. We can move package manager from Npm to Yarn for JavaScript now.
 
 |                                              | npm install   | npm ci  | yarn | 
 |----------------------------------------------|---------------|---------|------|
-| install without cache (without node_modules) | 61s           | 60s     | 46s|
-| install with cache (without node_modules)    | 58s           | 43s     | 36s|
-| install with cache (with node_modules)       | 10s           | 46s     | `2s`|
+| install without cache (without node_modules) | 61s           | 60s     | 46s  |
```

Comment:
update raw data?

Reply:
My bad, missed those. I'll update and push.